google-site-kit.php

```diff
@@ -29,6 +29,8 @@
 define( 'GOOGLESITEKIT_VERSION', '1.84.0' );
 define( 'GOOGLESITEKIT_PLUGIN_MAIN_FILE', __FILE__ );
 define( 'GOOGLESITEKIT_PHP_MINIMUM', '5.6.0' );
+define( 'GOOGLESITEKIT_WP_MINIMUM', '4.7.0' );
+define( 'GOOGLESITEKIT_WORDPRESS_VERSION', get_bloginfo( 'version' ) );
```

Comment:
We can already load the current WP version globally, we don't need to assign it to a constant. It isn't an expensive function call either, so let's call it where we need it.